Tests/NIOCoreTests/NIOAnyDebugTest.swift

```diff
@@ -24,7 +24,7 @@ class NIOAnyDebugTest: XCTestCase {
         let bb = ByteBuffer(string: "byte buffer string")
         XCTAssertTrue(
             wrappedInNIOAnyBlock(bb).contains(
-                "NIOAny { ByteBuffer { readerIndex: 0, writerIndex: 18, readableBytes: 18, capacity: 32, storageCapacity: 32, slice: _ByteBufferSlice { 0..<32 }, storage: "
+                "NIOAny { [627974652062756666657220737472696e67](18 bytes) }"
```

Comment:
Nit: I almost think with this change we should change NIOAny's `description` to include `ByteBuffer: ` when it's the `case .byteBuffer`. Previously that wasn't necessary because `ByteBuffer.description` contained the word `ByteBuffer` but that's now changed.

Reply:
Looking at `NIOAny` I would improve its description altogether to include information about the wrapped type. But I would do that in a separate issue/PR.